Sensus.sln

```diff
@@ -56,7 +56,10 @@ Global
 		Sensus.Shared\Sensus.Shared.projitems*{d33b6cdb-795c-4826-b131-47c526ace07f}*SharedItemsImports = 4
 	EndGlobalSection
 	GlobalSection(SolutionConfigurationPlatforms) = preSolution
+		Ad-Hoc|Android = Ad-Hoc|Android

```

Comment:
We need to figure out why VS is adding these ad-hoc configurations. Any thoughts?